bftengine/include/bftengine/ReplicaConfig.hpp

```diff
@@ -123,7 +123,12 @@ class ReplicaConfig : public concord::serialize::SerializableFactory<ReplicaConf
   // RSA public keys of all clients. Each public key holds set of distinct client (principal) ids which are expected to
   // sign with the matching private key
   std::set<std::pair<const std::string, std::set<uint16_t>>> publicKeysOfClients;
-
+  std::unordered_map<uint16_t, std::set<uint16_t>> clientGroups;
+  CONFIG_PARAM(clientsKeysPrefix, std::string, "", "the path to the client keys directory");
+  CONFIG_PARAM(saveClinetKeyFile,
```

Comment:
why not True?

Reply:
Basically, with Apollo and Hermes tests all replicas are looking on the same files.
Saving the same file from multiple sources may cause issues (at least the Apllo tests have failed)